Puppet/Puppet/AppDelegate.m

```diff
@@ -13,8 +14,11 @@
 #import "MobileCenterDistribute.h"
 #import "MobileCenterPush.h"
 #import "MSAlertController.h"
+#import "MSAnalyticsDelegate.h"
+#import "MSAnalyticsInternal.h"
+#import "MSEventLog.h"
 
-@interface AppDelegate () <MSCrashesDelegate, MSDistributeDelegate, MSPushDelegate>
+@interface AppDelegate () <MSCrashesDelegate, MSDistributeDelegate, MSPushDelegate, MSAnalyticsDelegate>
```

Comment:
Any reason of using MSAnalyticsDelegate? It is Xamarin only delegate and eventually we will remove or at least revise this one. This delegate is not a public delegate that can be used by applications.